core/src/test/scala/com/pingcap/tispark/telemetry/TelemetrySuite.scala

```diff
@@ -48,19 +43,19 @@ class TelemetrySuite extends SharedSQLContext {
     assert(resp.get.body.equals("test telemetry"))
   }
 
-  test("test telemetry") {
-    noException should be thrownBy {
-      val telemetry = new Telemetry
-      telemetry.setUrl("http://127.0.0.1:8091/test")
-      val teleMsg = new TeleMsg(_sparkSession)
-      telemetry.report(teleMsg)
-    }
-  }
-
   test("test get TiDB version") {
     val tiSparkTeleInfo = TiSparkTeleInfo.getTiSparkTeleInfo()
     assert(!tiSparkTeleInfo.get("tidb_version").contains("UNKNOWN"))
   }
+  test("test cacheInvalidateCallback does not work ") {
```

Comment:
what is this test for

Reply:
It not for this pr,It should be delete but I delete error code.